src/System.IO.Compression/src/MatchingRefApiCompatBaseline.txt

```diff
@@ -0,0 +1,3 @@
+Compat issues with assembly System.IO.Compression:
+TypesMustExist : Type 'System.IO.Compression.ZLibException' does not exist in the implementation but it does exist in the contract.
```

Comment:
@ViktorHofer should this type be exposed in the ref? If not is it exposed publicly in the implementation for serialization?

Reply:
The type was internal in netfx and Core < 2.0 but we had to make it public for serialization. It shouldn't be exposed in the ref. Seems correct.